share/completions/eselect.fish

```diff
@@ -17,8 +17,9 @@ function __fish_complete_eselect_action_options
     set -l parseregexp 's/^    ([a-zA-Z0-9_-]*)[ ]*/\1\t/g'
     set -l cmdl (commandline -poc)
 
-    # Disable further php completion
+    # Alter further php completion
     if [ (__fish_print_cmd_args_without_options)[2] = 'php' ]
+        eselect php list-modules | sed 's/ /\n/g'
```

Comment:
This sed should either be `string -a " " \n` (no quoting for the latter) or `tr " " \n` (here the quoting doesn't matter)